contracts/serverless-v2/GatewayJobs.sol

```diff
@@ -25,9 +25,20 @@ contract GatewayJobs is
     error GatewayJobsZeroAddressStakingToken();
     error GatewayJobsZeroAddressUsdcToken();
 
+    /**
+     * @notice Initializes the logic contract without any admins to safeguard against takeover of the logic contract.
+     * @param _stakingToken The staking token used in the system.
+     * @param _usdcToken The USDC token used for payments.
+     * @param _signMaxAge The maximum age of a valid signature in seconds.
+     * @param _relayBufferTime The buffer time allowed for relaying jobs.
+     * @param _executionFeePerMs The execution fee per millisecond.
+     * @param _slashCompForGateway The amount to be slashed from a gateway.
```

Comment:
its not from gateway, it is to the gateway when executors failed to complete the request.